job-dsls/jobs/tools/tools_agent_image_build_PR_test.groovy

```diff
@@ -56,8 +56,9 @@ job(jobName) {
     parameters {
         stringParam ("OWNER","kiegroup","")
         stringParam ("BRANCH","main","")
-        stringParam ("PACKER_VAR_FILE","packer-kie-rhel7-vars.json","")
-
+        stringParam ("BXMS_JENKINS_BRANCH","master","")
```

Comment:
master or main branch?

Reply:
This is the gerrit bxms-jenkins repo still using master.

Reply:
ok